src/renderer/widgets/WalletDetails/ui/WalletConnectDetails.tsx

```diff
@@ -115,6 +118,45 @@ export const WalletConnectDetails = ({ wallet, accounts, onClose }: Props) => {
     </DropdownIconButton>
   );
 
+  const tabItems: TabItem[] = [
+    {
+      id: 'accounts',
+      title: t('walletDetails.common.accountTabTitle'),
+      panel: (
+        <>
+          {wcDetailsUtils.isNotStarted(reconnectStep, wallet.isConnected) && (
+            <MultiAccountsList accounts={accountsList} className="h-[367px]" headerClassName="pt-4 pb-2" />
+          )}
+
+          {wcDetailsUtils.isReadyToReconnect(reconnectStep, wallet.isConnected) && (
+            <div className="flex flex-col h-[404px] justify-center items-center">
+              <Icon name="document" size={64} className="mb-6 text-icon-default" />
+              <SmallTitleText className="mb-2">{t('walletDetails.walletConnect.disconnectedTitle')}</SmallTitleText>
+              <FootnoteText className="mb-4 text-text-tertiary">
+                {t('walletDetails.walletConnect.disconnectedDescription')}
+              </FootnoteText>
+              <Button onClick={showReconnectConfirm}>{t('walletDetails.walletConnect.reconnectButton')}</Button>
+            </div>
+          )}
+
+          {wcDetailsUtils.isReconnecting(reconnectStep) && (
+            <div className="flex flex-col h-[409px] justify-center items-center">
+              <video className="object-contain h-[409px]" autoPlay loop>
+                <source src={wallet_connect_reconnect_webm} type="video/webm" />
+                <source src={wallet_connect_reconnect} type="video/mp4" />
+              </video>
+            </div>
+          )}
+        </>
+      ),
+    },
+    {
+      id: 'proxies',
+      title: t('walletDetails.common.proxiesTabTitle'),
+      panel: <ProxiesList walletId={wallet.id} className="h-[385px] mt-6" />,
```

Comment:
proxies list has different height and top margin everywhere, is it correct?

Reply:
That would be nice but I think that's too much work for that PR cause every wallet details modal made from different components and they use in other places too, so it would take couple hours at least to refactor all modal and change all place where this components are used